osmaxx/core/templates/pages/about_us.html

```diff
@@ -24,5 +25,23 @@
         Postfach 1475<br />
         CH-8640 Rapperswil<br />
     </p>
+
+    <h3>Data Information</h3>
+    <h4>Base Data</h4>
+    <p>
+        All the base data used by OSMaxx stems originally from
+        <em>OSM</em> and can be used under their terms.
+    </p>
```

Comment:
Done partially in 838c24d2a330ed5517d688925d23355e8b6463f7
let's add the "License" section later